policy/modules/system/init.te

```diff
@@ -35,6 +35,7 @@ attribute init_path_unit_loc_type;
 attribute init_script_domain_type;
 attribute init_script_file_type;
 attribute init_run_all_scripts_domain;
+attribute init_linkable_keyring_type;
```

Comment:
Where is this going to be used? I see an interface but no callers.

Reply:
The interface `init_linkable_keyring` which grants this attribute is called [here](https://github.com/SELinuxProject/refpolicy/pull/357/files#diff-7ab643cade2410f2faebd8a6aae45f87bd21a81239156731fb6937c33f566815R74) by systemd user instances.

Reply:
Github doesn't make it easy to see these I suppose. Both this and the below interface are called in `systemd_role_template`.